app/controllers/metadata_controller.rb

```diff
@@ -21,14 +20,31 @@ def create
     if row_errors > 0
       feedback = "Your upload has finished processing. #{rows} works were updated successfully; #{row_errors} rows encountered errors. Download the error file here: #{link}"
     else
-      feedback = "Your upload has finished processing. #{rows} works were updated successfully."      
+      feedback = "Your upload has finished processing. #{rows} works were updated successfully."
     end
 
     flash[:alert] = feedback
 
     ajax_redirect_to edit_look_collection_path(collection.owner, collection)
   end
 
+  def refresh
+    collection = Collection.find(params[:id])
+
+    # Make sure import logs folder exists
+    Dir.mkdir("#{Rails.root}/public/metadata") unless Dir.exist?("#{Rails.root}/public/metadata")
+
+    # Create logfile for collection
+    log_file = "#{Rails.root}/public/metadata/#{Time.current.to_i}_refresh_collection_#{collection.id}.log"
```

Comment:
Can we rename this so that
* It's in a subdirectory of `public/metadata`, since we're likely to want statically-generated public metadata for other things (`public/metadata/refresh/log` would be OK)
* The collection ID precedes the current time in the filename, so that logs for the same collection will be grouped by `ls`